From ltsi-dev-bounces@lists.linuxfoundation.org Tue Jul 10 22:13:24 2012
From: "Shimoda, Yoshihiro" <yoshihiro.shimoda.uh@renesas.com>
Date: Wed, 11 Jul 2012 14:13:20 +0900
Subject: sh: also without PM_RUNTIME pm_runtime.o must be built
To: ltsi-dev@lists.linuxfoundation.org
Message-ID: <4FFD0B70.1010608@renesas.com>


>From c219336ade79e8ce6f70b527c20391a239278ce3 Mon Sep 17 00:00:00 2001
From: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Date: Tue, 10 Jan 2012 16:04:11 +0100
Subject: sh: also without PM_RUNTIME pm_runtime.o must be built

When CONFIG_PM_RUNTIME is off, drivers/sh/pm_runtime.o still has to be
built on sh platforms, because then it provides means to statically
switch on device PM clocks.

Signed-off-by: Guennadi Liakhovetski <g.liakhovetski@gmx.de>
Signed-off-by: Paul Mundt <lethal@linux-sh.org>
(cherry picked from commit d3442b2bb7ce4089fadcf483b8ae9766ab480238)
Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
Tested-by: Simon Horman <horms@verge.net.au>
---
 drivers/sh/Makefile |    9 +--------
 1 file changed, 1 insertion(+), 8 deletions(-)

--- a/drivers/sh/Makefile
+++ b/drivers/sh/Makefile
@@ -7,11 +7,4 @@ obj-$(CONFIG_HAVE_CLK)		+= clk/
 obj-$(CONFIG_MAPLE)		+= maple/
 obj-$(CONFIG_SUPERHYWAY)	+= superhyway/
 obj-$(CONFIG_GENERIC_GPIO)	+= pfc.o
-
-#
-# For the moment we only use this framework for ARM-based SH/R-Mobile
-# platforms and generic SH. SH-based SH-Mobile platforms are still using
-# an older framework that is pending up-porting, at which point this
-# special casing can go away.
-#
-obj-$(CONFIG_SUPERH)$(CONFIG_ARCH_SHMOBILE)	+= pm_runtime.o
+obj-y				+= pm_runtime.o
